Sources/Controllers/Map/OAMapHudViewController.mm

```diff
@@ -1474,6 +1496,34 @@ - (void) updateMapSettingsButton
     _mapSettingsButton.tintColorDay = UIColorFromRGB(mode.getIconColor);
     _mapSettingsButton.tintColorNight = UIColorFromRGB(mode.getIconColor);
     [_mapSettingsButton updateColorsForPressedState:NO];
+    
+    NSString *stringKey = _settings.applicationMode.get.stringKey;
```

Comment:
Extract to a method and remove settings from class fields since you use it only once.